promotions/spec/models/concerns/conditions/taxon_condition_spec.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
```

Comment:
same here. stores will also benefit from an shared example for their custom conditions

Reply:
thanks